src/services/LiquidityService.sol

```diff
@@ -421,25 +422,18 @@ contract LiquidityService is OwnableUpgradeable, ReentrancyGuardUpgradeable, ILi
       revert LiquidityService_InsufficientLiquidityBuffer();
     }
 
-    ConfigStorage.LiquidityConfig memory _liquidityConfig = _configStorage.getLiquidityConfig();
-    PerpStorage.GlobalState memory _globalState = PerpStorage(perpStorage).getGlobalState();
     Calculator _calculator = Calculator(_configStorage.calculator());
 
     // Validate Max HLP Utilization
     // =====================================
-    // reserveValue / HLP TVL > maxHLPUtilization
-    // Transform to save precision:
-    // reserveValue > maxHLPUtilization * HLP TVL
+    // There should be at lest 2x of Global PnL left in HLP to allow traders to settle their trades
+
     uint256 hlpTVL = _calculator.getHLPValueE30(false);
 
-    if (_globalState.reserveValueE30 * BPS > _liquidityConfig.maxHLPUtilizationBPS * hlpTVL) {
+    uint256 _globalPnL = HMXLib.abs(_calculator.getGlobalPNLE30());
```

Comment:
This shouldn't take `abs`, should take consideration only when HLP is loss. If HLP is profit, should be fine.